scripts/diagnose.sh

```diff
@@ -144,6 +144,11 @@ commands=(
 	'top -b -n 1'
 	'grep -vE \"/var/cache/ldconfig/aux-cache|md5sum|/etc/hostname|/etc/machine-id|/etc/resin-supervisor/supervisor.conf|/etc/systemd/timesyncd.conf|/home/root/.rnd\" /resinos.fingerprint | md5sum --quiet -c ' # https://github.com/balena-os/meta-balena/issues/1618
 
+	# Boot performance
+	'echo === Boot ==='
```

Comment:
```suggestion
	'echo === BOOT ==='
```

for consistency. i'd also rather put this near the top, as i try to keep the sections alphabetical